crates/uv/tests/cache_prune.rs

```diff
@@ -102,25 +97,14 @@ fn prune_stale_directory() -> Result<()> {
     let simple = context.cache_dir.child("simple-v4");
     simple.create_dir_all()?;
 
-    let filters = [
-        (r"Pruning cache at: .*", "Pruning cache at: [CACHE_DIR]"),
-        (
-            r"Removing dangling cache entry: .*[\\|/]simple-v4",
-            "Pruning cache at: [CACHE_DIR]/simple-v4",
-        ),
-    ]
-    .into_iter()
-    .chain(INSTA_FILTERS.to_vec())
-    .collect::<Vec<_>>();
-
-    uv_snapshot!(filters, prune_command(&context).arg("--verbose"), @r###"
+    uv_snapshot!(context.filters(), prune_command(&context).arg("--verbose"), @r###"
     success: true
     exit_code: 0
     ----- stdout -----
 
     ----- stderr -----
-    Pruning cache at: [CACHE_DIR]
-    DEBUG Pruning cache at: [CACHE_DIR]/simple-v4
+    Pruning cache at: [CACHE_DIR]/
+    DEBUG Removing dangling cache entry: [CACHE_DIR]/simple-v4
```

Comment:
This is a great example of a non-standard filter resulting in an incorrect snapshot.